javascript/javascript3/week3/scripts/quizToSaveApi.js

```diff
@@ -0,0 +1,73 @@
+
+
+const owner = "jalilhu";
+
+const repo = "Quiz_App";
+const path = "userQuestions.json"; // Path to the JSON file
+const token = ""
```

Comment:
I like the idea of being able to save updates to your questions! That being said, this will unfortunately not work. JavaScript files are visible to the browser and the user using that browser. You access token for Github is privat and needs to remain private. Hence, you can never publish a website that directly talks to Github from the JS file that way. However, you can use a server to accomplish that goal. Then the server will know the secret and check if a user has access to the specific action and forward it to the Github API.